src/components/form/OneClickForm/constants/credentialTypes.ts

```diff
@@ -0,0 +1,7 @@
+// Not all credential type are here because is not needed to, but can be added if so.
```

Comment:
Why are these needed here and others are not? Please include the reason in the comment.